rasa/nlu/classifiers/__init__.py

```diff
@@ -1,3 +1,23 @@
 # How many labels are at max put into the output
 # ranking, everything else will be cut off
 LABEL_RANKING_LENGTH = 10
+
+import scipy.sparse
+
+
+# TODO should be removed in next PR
+def convert_sparse_back(sparse_features: scipy.sparse.csr_matrix):
```

Comment:
only a temporary function, that should be removed in the next PR, so I guess naming is not that important, just added it so that the tests are passing